### ISSUE TITLE
Offline deployment broken for v93

### ISSUE THREAD

**Issue by metron2** [cloudfoundry/bosh-vsphere-cpi-release#366]
**Describe the bug**
I'm trying to run create-env offline but the vsphere_cpi package doesn't compile. It looks like bundler is trying to fetch a missing dependency.

**To Reproduce**
1. Run create-env without the internet

**CPI Error Log**
fetcher.rb is 3rd in stack

**Expected behavior**
Ruby 3.1 should not be fetching anything

**Release Version & Related Info (please complete the following information):**
 -v93
 - https://github.com/cloudfoundry/bosh-deployment/commit/f0cb03c59c1d2428f4ab87a6e415dc0a9546be0b
 - 1.125 vsphere esxi ubuntu jammy

**Comment by jpalermo** [comment]
Hi @metron2 

Yeah, we thought this was an issue due to the divergence of the `BUNDLED WITH` in the Gemfile.lock and the actual bundler version included in the ruby release package.

However we then found [this commit](https://github.com/cloudfoundry/bosh-vsphere-cpi-release/commit/e5d323a868dd8b597aca24b084b480a58edafb80) from last year that should prevent bundler from trying to installed the `BUNDLED WITH` version at runtime (which does not work in offline environments).

So we're not totally sure why the problem is happening...

We have implemented some standardized auto-bumping of ruby which should keep the versions in sync and avoid the problem. That will be released in v94, probably today or tomorrow.

In the future we would like to add some offline environments to our testing, but haven't had the time to prioritize the creation of those.
